vendor/github.com/openshift/installer/pkg/aro/aroign/etchosts.go

```diff
@@ -0,0 +1,73 @@
+package aroign
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/blang/semver"
+	ign3types "github.com/coreos/ignition/v2/config/v3_2/types"
+	"github.com/vincent-petithory/dataurl"
+)
+
+const (
+	configFileName = "etchosts.conf"
+)
+
+func config(clusterDomain string, apiIntIP string, gatewayDomains []string, gatewayPrivateEndpointIP string) ([]byte, error) {
+	t := template.Must(template.New(configFileName).Parse(`{{ .APIIntIP }}	api.{{ .ClusterDomain }} api-int.{{ .ClusterDomain }}
```

Comment:
This overwrites /etc/hosts rather than appends, correct? Include the original content of /etc/hosts here:
```
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
```

Reply:
I don't think so, see line 62.